Jenkinsfile

```diff
@@ -32,6 +32,12 @@ pipeline {
                         sh './tests/scripts/functional-tests-ams.sh'
                     }
                 }
+
+                stage('functional tests model posprocessing') {
```

Comment:
Why another stage? Can't we do such things along with other functional tests for AMS?

Reply:
This is postprocessing functional testing not whole end to end. Lets put it in different stage.